examples/hello-compute/shader.wgsl

```diff
@@ -23,6 +23,11 @@ fn collatz_iterations(n_base: u32) -> u32{
             n = n / 2u;
         }
         else {
+            // Overflow? (i.e. 3*n + 1 > 0xffffffffu?)
+            if (n >= 1431655765u) {   // 0x55555555u
+                return 4294967295u;   // 0xffffffffu
```

Comment:
we could write this simpler, i.e. `~0u` or `u32(-1)`

Reply:
`~0u` would have been nice! I'll update this when hex is working to `0xffffffff` to emphasize it is the max value and to make it consistent with the Rust constant.